lib/processLayerStyles.js

```diff
@@ -72,12 +75,62 @@ const layerFill = (layerStyle) => {
 
         }
 
+        if (fill.fillType == 4 && fill.isEnabled == true) { // Images
```

Comment:
Lol, love a good Magic Number :) What are the other fill types - do we need to add them to a list of future features to implement?

Reply:
@cdemetriadis do you know what the missing numbers are for fill types?